spec/support/api_stubs/dttp/provider.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module ApiStubs
+  module Dttp
+    module Provider
+      def self.attributes
+        {
+          "name" => "Test Organisation",
+          "dfe_ukprn" => "A123Q",
+          "accountid" => SecureRandom.uuid,
+        }
```

Comment:
https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#custom-strategies

can we consider using this?

Reply:
Adding in a card for this: https://trello.com/c/bZJuqovc/1184-consider-using-factorybot-custom-strategies-instead-of-api-stubs